backend/medtagger/repositories/scan_categories.py

```diff
@@ -39,3 +39,27 @@ def add_new_category(key: str, name: str, image_path: str) -> ScanCategory:
             category = ScanCategory(key, name, image_path)
             session.add(category)
         return category
+
+    @staticmethod
+    def assign_label_tag(tag: LabelTag, scan_category_key: str) -> None:
```

Comment:
This can be done a lot easier.

**Example:**
```python
scan_category = ScanCategory.query.filter(Scancategory.key == key).one()
scan_category.available_tags.append(tag)
scan_category.save()
```

Unassigning can be done in the same way but you should remove element from list.